docs/workloads/azure/data/getting_started/de_shared_resourcedeployment_azure.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
General remark: please make sure that docs in stacks-azure-data are consistent with these (make sure that you run  pre-commit hooks in that repo to check/fix linting)

Reply:
yeah sure